README.md

```diff
@@ -22,7 +22,8 @@
 
 service    | Kinetic | Melodic | Master
 ---------- | ------- | ------- | ------
-Travis     | [![Build Status](https://travis-ci.com/ros-planning/moveit.svg?branch=kinetic-devel)](https://travis-ci.com/ros-planning/moveit/branches) | [![Build Status](https://travis-ci.com/ros-planning/moveit.svg?branch=melodic-devel)](https://travis-ci.com/ros-planning/moveit/branches) | [![Build Status](https://travis-ci.com/ros-planning/moveit.svg?branch=master)](https://travis-ci.com/ros-planning/moveit/branches) |
+GitHub | [![Format](https://github.com/ros-planning/moveit/actions/workflows/format.yml/badge.svg?branch=kinetic-devel)](https://github.com/ros-planning/moveit/actions/workflows/format.yml?branch=kinetic-devel) [![BuildAndTest](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml/badge.svg?branch=kinetic-devel)](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml?branch=kinetic-devel) | [![Format](https://github.com/ros-planning/moveit/actions/workflows/format.yml/badge.svg?branch=melodic-devel)](https://github.com/ros-planning/moveit/actions/workflows/format.yml?branch=melodic-devel) [![BuildAndTest](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml/badge.svg?branch=melodic-devel)](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml?branch=melodic-devel) | [![Format](https://github.com/ros-planning/moveit/actions/workflows/format.yml/badge.svg?branch=master)](https://github.com/ros-planning/moveit/actions/workflows/format.yml?branch=master) [![BuildAndTest](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml/badge.svg?branch=master)](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml?branch=master) |
```

Comment:
You could add the Noetic build for the master column?
https://build.ros.org/job/Ndev__moveit__ubuntu_focal_amd64/

Reply:
done!